### PR TITLE
fix: rename likertScale PCI to likert in order to be able to import and use it straight away

### PR DESCRIPTION
**Pull request by wazelin**
Follow up to #394.

Changing the typeIdentifier to a new one (`likertScoreInteraction`) would allow us to provide the PCI to OECD by manual upload, whereas keeping the original typeIdentifier (`likertScaleInteraction`) would require a deployment (and backport to finalised March release).

### CODE REVIEW

File: views/js/pciCreator/ims/likertScaleInteraction/creator/tpl/markup.tpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="likertScaleInteraction">
+<div class="likertInteraction">
```
Comment (oatymart) [inline views/js/pciCreator/ims/likertScaleInteraction/creator/tpl/markup.tpl:1]: This identifier is already taken by `pciCreator/ims/likertConfig`. (That's _another_ version of the PCI, which never had an install script. I have no idea which environments it was ever added to. Also its version is 1.2.0)

If using this PR, we should either choose a brand new identifier, or delete `likertConfig`.

File: views/js/pciCreator/ims/likertScaleInteraction/runtime/css/likerInteraction.css
Comment (oatymart) [inline views/js/pciCreator/ims/likertScaleInteraction/runtime/css/likerInteraction.css:1]: New file name is missing a `t`. I guess it won't be loadable?

**Review by oatymart — CHANGES_REQUESTED** [review]
If creating this PCI to be distinct from the `dev/likertScaleInteraction`, we need to pick a properly unique identifier.

Also, we are missing a PHP registration script here. Was that intended?

File: views/js/pciCreator/ims/likertScaleInteraction/runtime/css/likerInteraction.css
Comment (wazelin) [reply views/js/pciCreator/ims/likertScaleInteraction/runtime/css/likerInteraction.css:1]: Updated the PR reducing the number of changes.

Comment (oatymart) [reply views/js/pciCreator/ims/likertScaleInteraction/runtime/css/likerInteraction.css:1]: Sorry, but the simple identifier renaming was not really enough. I've changed the folder names, and the tests (all references from `Scale` -> `Score`) to leave a place that is not unpleasant to work in.

I've also updated the installer & migration.

The good news is, it still works fine as either a fresh install, a TAO update, or a zip upload.

**Comment by wazelin** [comment]
> Also, we are missing a PHP registration script here. Was that intended?

To be fair, I thought more of _replacing_ the old likertScale PCI with you new version. Just under a different ID.

Not sure that we actually need both of them in the codebase.

Hence the lack of a dedicated registration script.

But I can see you've already covered that bit. Clean job, as always! Thanks!

**Comment by oatymart** [comment]
I re-tested the import of the zipped PCI from my original PR, and got it to work in a develop TAO. (The 15-minute FE cache was making me think I had errors before!)

So I'm fairly convinced we don't need this identifier change any more.

**Comment by wazelin** [comment]
> we don't need this identifier change any more

That's nice. However, there's another problem we're working around with this change. This piece of code will still continue blocking the PCI in item authoring on the customer's environment.

https://github.com/oat-sa/extension-tao-deliver-connect/blob/master/model/FeatureFlag/FeatureFlagClientConfigHandler.php#L43

**Comment by oatymart** [comment]
The identifier change was committed directly to #394

**Pull request closed without merging**